gnovm/pkg/gnolang/types.go

```diff
@@ -2090,6 +2262,7 @@ func assertSameTypes(lt, rt Type) {
 	} else if lt.TypeID() == rt.TypeID() {
 		// non-nil types are identical.
 	} else {
+		panic("panic assertSameTypes")
```

Comment:
maybe this should go after the next line

Reply:
code removed.